docs/working-drafts/issue-29-erratum.md.adoc

```diff
@@ -0,0 +1,135 @@
+// please start a new sentence on a new line.
+// this facilitates commentary on this document using code-review tools.
+
+= Issue 29 - dfdl:prefixLengthKind Simple Type having DFDL Statement Annotations
+
+
+Sections of the DFDL v1.0 Spec which will require updating include:
+
+## Section 9.5 - Evaluation Order for Statement Annotations
+
+An element (or element reference) having lengthKind 'prefixed' can
+have statements associated with its prefix length type. This is
+discussed below in Section (new sub-section of 9.5).
+
+A new sub-section is added: Evaluation Order for Statement Annotations
+for Elements with dfdl:lengthKind 'prefixed'.
+
+This new sub-section contains the content below.
+
+When an element or element reference has lengthKind 'prefixed' there
+can be additional statement annotations associated with the prefix
+length type.
+
+Definition: the resolved set of statement annotations for the simple
+type that is identified by the dfdl:prefixLengthKind property are
+called the _prefix statement annotations_.
+
+Definition: the resolve set of statement annotations for the nested
+prefix simple type that is identified by the dfdl:prefixLengthKind
+property on a simple type itself having lengthKind 'prefixed', are
+called the _prefix-prefix statement annotations_.
+
+For statement annotations associated with any prefix length, any DFDL
+expressions evaluated cannot refer to the DFDL Infoset using path
+expressions.
+
+The value of the prefix (or prefix-prefix) itself can be referenced
+using the expression ".".
+
+When parsing the value of the "." expression for a prefix (or
+prefix-prefix) quasi-element is as before any adjustment (subtraction)
+from the value based on dfdl:prefixIncludesPrefixLength.
+
+When unparsing the value of the "." expression for a prefix (or
+prefix-prefix) quasi-element is as after any adjustment (addition) to
+the value based on dfdl:prefixIncludesPrefixLength.
+
+The evaluation order of statements associated with prefix lengths is:
+
+1. prefix-prefix statement annotations
```

Comment:
I don't follow this. It talks only about statement annotations but I don't think you can do that. The prefix and prefix-prefix also have a format annotation.  We need to be clear on the order of evaluation of **all** annotations that apply to an element with a prefix length. Did you mean to say "all annotations"?

Reply:
I attached a zip file with the example in it to the issue ticket here: 

https://github.com/OpenGridForum/DFDL/issues/29#issuecomment-1516996502

I updated the wording to say all annotations.